common/containers/Tabs/GenerateWallet/components/GenerateWalletPasswordComponent.jsx

```diff
@@ -0,0 +1,145 @@
+import React, {Component} from "react";
+import PropTypes from "prop-types";
+
+import {Field, reduxForm} from "redux-form";
+import GenerateWalletPasswordInputComponent from "./GenerateWalletPasswordInputComponent";
+import LedgerTrezorWarning from "./LedgerTrezorWarning";
+
+
+// VALIDATORS
+const minLength = min => value => {
+    return value && value.length < min ? `Must be ${min} characters or more` : undefined
+}
+const minLength9 = minLength(9)
+const required = value => value ? undefined : 'Required'
+
+
+class GenerateWalletPasswordComponent extends Component {
+    constructor(props) {
+        super(props)
+    }
+
+    static propTypes = {
+        title: PropTypes.string,
+        body: PropTypes.string,
+        userId: PropTypes.number,
+        id: PropTypes.number,
+        generateWalletPassword: PropTypes.object,
+        showPassword: PropTypes.bool,
+        showGenerateWalletPasswordAction: PropTypes.func,
+        generateWalletFileAction: PropTypes.func,
+        generateWalletHasDownloadedFileAction: PropTypes.func,
+        generateWalletFile: PropTypes.bool,
+        hasDownloadedWalletFile: PropTypes.bool,
+        generateWalletContinueToPaperAction: PropTypes.func,
+        canProceedToPaper: PropTypes.bool
+    };
+
+
+    continueToPaper() {
+    }
+
+    downloaded() {
+        let nextState = this.state
+        nextState.hasDownloadedWalletFile = true
+        this.setState(nextState)
+    }
+
+
+    render() {
+        const {
+            // handleSubmit,
+            // pristine,
+            // reset,
+            // submitting,
+            generateWalletPassword,
+            showPassword,
```

Comment:
Do we really need this in store? Along with actions to toggle that that multiply every time you have that control :)